### PR TITLE
Aborting undeploy if there are instances more than the limit

### PR DESCRIPTION
**Pull request by vinok88**
Aborting undeploy if there are too much instances. Issue - https://wso2.org/jira/browse/BPS-363

### CODE REVIEW

File: components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/store/BPELUIException.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wso2.carbon.bpel.core.ode.integration.store;
+
+/**
+ * Throws when there are BPEL UI related problem occurred.
+ */
+public class BPELUIException extends RuntimeException {
```
Comment (milindaperera) [inline components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/store/BPELUIException.java]: Use Exception instead RuntimeException

File: components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/store/BPELUIException.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    public BPELUIException(String s) {
```
Comment (milindaperera) [inline components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/store/BPELUIException.java]: pass the exception object also
eg: BPMNException(String message, Exception ex)

File: components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/store/BPELUIException.java (hunk 1 above)
Comment (vinok88) [inline components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/store/BPELUIException.java]: Fixed in https://github.com/wso2/carbon-business-process/pull/176.

File: components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/store/BPELUIException.java (hunk 2 above)
Comment (vinok88) [inline components/bpel/org.wso2.carbon.bpel/src/main/java/org/wso2/carbon/bpel/core/ode/integration/store/BPELUIException.java]: Fixed in https://github.com/wso2/carbon-business-process/pull/176.

**Pull request merged**